app/main.py

```diff
@@ -1,2 +1,104 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any, Hashable
+
+
+@dataclass
+class DictItem:
+    key: Hashable
+    value: Any
+
+
+class DictIterator:
+    def __init__(self, dct: Dictionary) -> None:
+        self.box = iter(dct.storage)
+
+    def __next__(self) -> Any:
+        result = next(self.box)
+        while result is None:
+            result = next(self.box)
+        return result.key
+
+
 class Dictionary:
-    pass
+    _capacity = 8
+    _load_factor = 2 / 3
+    _threshold = 5
+
+    def __init__(self) -> None:
+        self._size = 0
+        self.storage = [None for _ in range(self._capacity)]
+
+    def get(self, key: Hashable, default: Any = None) -> Any:
+        try:
+            return self[key]
```

Comment:
let me understand: do your values on `self.storage`? So what are you doing here?

Reply:
You call magic method `__getitem__` and if you haven't key in dict, function `get` will return `None`, no exception `KeyError`